.github/workflows/scale-test-egw.yaml

```diff
@@ -0,0 +1,485 @@
+    }}
+  cancel-in-progress: true
+
+env:
+  # renovate: datasource=golang-version depName=go
+  go_version: 1.22.4
+  # renovate: datasource=github-releases depName=eksctl-io/eksctl
+  eksctl_version: v0.183.0
+  # renovate: datasource=github-releases depName=kubernetes/kubernetes
+  kubectl_version: v1.30.2
+  # renovate: datasource=docker depName=google/cloud-sdk
+  gcloud_version: 485.0.0
+  # renovate: datasource=git-refs depName=https://github.com/cilium/scaffolding branch=main
+  egw_utils_ref: d0e4d60cf847b37fbde8e9ed4a11c70d1ae349ef
+
+  test_name: egw
+  cluster_name: ${{ github.run_id }}-${{ github.run_attempt }}
+  eks_region: us-west-1
+  k8s_version: 1.29
+  AWS_SSH_KEY: ~/.ssh/id_aws.pub
+  AWS_SSH_KEY_PRIVATE: ~/.ssh/id_aws
+
+jobs:
+  echo-inputs:
+    if: ${{ github.event_name == 'workflow_dispatch' }}
+    name: Echo Workflow Dispatch Inputs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Echo Workflow Dispatch Inputs
+        run: |
+          echo '${{ tojson(inputs) }}'
+
+  commit-status-start:
+    name: Commit Status Start
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set initial commit status
+        uses: myrotvorets/set-commit-status-action@3730c0a348a2ace3c110851bed53331bc6406e9f # v2.0.1
+        with:
+          sha: ${{ inputs.SHA || github.sha }}
+
+  install-and-scaletest:
+    runs-on: ubuntu-latest
+    name: Install and Scale Test
+    timeout-minutes: 150
+    strategy:
+      matrix:
+        test_type:
+          - md-bs # Abbreviated "masquerade delay - baseline"
+          - md
+    steps:
+      - name: Checkout context ref (trusted)
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          ref: ${{ inputs.context-ref || github.sha }}
+          persist-credentials: false
+
+      - name: Set Environment Variables
+        uses: ./.github/actions/set-env-variables
+
+      - name: Set up job variables
+        id: vars
+        run: |
+          if [ "${{ github.event_name }}" = "workflow_dispatch" ] ; then
+            SHA="${{ inputs.SHA }}"
+          else
+            SHA="${{ github.sha }}"
+          fi
+
+          CILIUM_INSTALL_DEFAULTS="--chart-directory=install/kubernetes/cilium \
+            --set pprof.enabled=true \
+            --helm-set=prometheus.enabled=true \
+            --helm-set=cluster.name=${{ env.cluster_name }} \
+            --helm-set=egressGateway.enabled=true \
+            --helm-set=bpf.masquerade=true \
+            --helm-set=kubeProxyReplacement=true \
+            --helm-set=l7Proxy=false \
+            --helm-set=egressMasqueradeInterfaces="" \
+            --wait=false \
+            --nodes-without-cilium \
+            --helm-set=eni.enabled=true \
+            --helm-set=ipam.mode=eni \
+            --helm-set=eni.awsEnablePrefixDelegation=true"
+
+          # only add SHA to the image tags if it was set
+          if [ -n "${SHA}" ]; then
```

Comment:
Nit. This check could be likely dropped, as the workflow always expects the SHA to be specified.

Reply:
Good point, thanks.